### PR TITLE
docs(core): :memo: update documentation for CoreCircularProgress

### PR DESCRIPTION
**Pull request by PritamIT2023**
update documentation for CoreCircularProgress

Ref: #201

## Description

Clearly describe the changes you've made in this pull request. Explain the purpose and reasoning behind the changes. Reference any relevant issues or discussions using keywords like "Fixes #<issue_number>" or "Resolves #<issue_number>".

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: app/components/component-docs/feedback/CoreCircularProgress.docs.js [hunk 1]
```diff
@@ -174,23 +232,45 @@ export default function CoreCircularProgressDocs() {
...
+                </>
+              }
+            />
+
           </>
         }
       />
 
-      <ComponentProps component={CoreCircularProgress} />
+      <CoreH4>CoreCircularProgress</CoreH4>
```
Comment (techoneel) [inline app/components/component-docs/feedback/CoreCircularProgress.docs.js]: remove this line

File: app/components/component-docs/feedback/CoreCircularProgress.docs.js [hunk 2]
```diff
@@ -174,23 +232,45 @@ export default function CoreCircularProgressDocs() {
...
+
+      <CodeImport name="CoreCircularProgress" />
```
Comment (techoneel) [inline app/components/component-docs/feedback/CoreCircularProgress.docs.js]: remove this line

**Review by techoneel — DISMISSED** [review]
please do the changes needful

File: app/components/component-docs/feedback/CoreCircularProgress.docs.js (hunk 1 above)
Comment (PritamIT2023) [reply app/components/component-docs/feedback/CoreCircularProgress.docs.js]: Removed that unnecessary line of code

File: app/components/component-docs/feedback/CoreCircularProgress.docs.js (hunk 2 above)
Comment (PritamIT2023) [reply app/components/component-docs/feedback/CoreCircularProgress.docs.js]: Removed that unnecessary line of code

**Review by techoneel — APPROVED** [review]
Good to go

**Pull request merged**